WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -157,26 +158,26 @@ public void bindView(final View view, Context context, Cursor cursor) {
             view.setTag(holder);
         }
 
-        final String mediaId = cursor.getString(cursor.getColumnIndex("mediaId"));
+        final String mediaId = cursor.getString(cursor.getColumnIndex(WordPressDB.COLUMN_NAME_MEDIA_ID));
```

Comment:
I didn't get rid of all the string literals, but it's a good start :checkered_flag: